lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_merge_translators_strings.rb

```diff
@@ -24,7 +24,7 @@ def self.merge_folder(strings_folder)
         tmp_main_file = main_file + '.tmp'
         FileUtils.cp(main_file, tmp_main_file)
 
-        join_files = Dir.glob(File.join("#{strings_folder}", 'strings-*.xml'))
+        join_files = Dir.glob(File.join(strings_folder.to_s, 'strings-*.xml'))
```

Comment:
Unlike above, it's not clear whether it's safe to assume `strings_folder` will always be a string. I'm going to leave the suggestion anyways, might not be safe, but it's reasonable.

```suggestion
        join_files = Dir.glob(File.join(strings_folder, 'strings-*.xml'))
```